lfucg-exactions/server/accounts/helper.py

```diff
@@ -39,3 +42,42 @@ def send_lost_username_email(user):
     msg = EmailMultiAlternatives(subject, text_content, from_email, [user.email])
     msg.attach_alternative(html_content, "text/html")
     msg.send()
+
+def send_email_to_supervisors(**kwargs):
+    if 'perm_name' in kwargs:
+        perm = Permission.objects.get(codename=kwargs['perm_name'])
+
+    q_objects = Q()
+    for group in kwargs['group']:
+        q_objects.add(Q(groups__name=group), Q.OR)
+    users = User.objects.filter(Q(profile__is_supervisor=True), q_objects)
+
+    to_emails = []
+    for user in users:
+        to_emails.append(user.email)
```

Comment:
same: to_emails = list(users)

Reply:
@Wei-KeatKong - how would you just get the emails? This is taking the usernames if I do it this way.

Reply:
sorry, do list(users.values_list('email', flat=True))